panic on "jump to owner" of reflect.Value.Elem on zero Value

<img src="https://raw.githubusercontent.com/derailed/k9s/master/assets/k9s_err.png" align="right" width="100" height="auto"/>

<br/>
<br/>
<br/>

**Describe the bug**
Using Shift+j keyboard shortcut for jump to owner, I run into a panic of:
```
panic: reflect: call of reflect.Value.Elem on zero Value                                                                   
                                                                                                                                                             
                                                                                          goroutine 221 [running]:
                                                                                                                  reflect.Value.Elem({0x0?, 0x0?, )
  	reflect/value.go:1254
                                    github.com/derailed/k9s/internal/render.hydrate({0x0, 0x0}, {}, {0x14002055ef0, 0x5,)
                                   	github.com/derailed/k9s/internal/render/cust_cols.go:189 +0x50c
                                                                                                       github.com/derailed/k9s/internal/render.ColumnSpecs.realize
                                                                                     	github.com/derailed/k9s/internal/render/cust_cols.go:130
                                                                                                                                                       github.com/derailed/k9s/internal/render.(*Table).Render
                                                                                                                                      	github.com/derailed/k9s/internal/render/table.go:98
                                    github.com/derailed/k9s/internal/model1.GenericHydrate.func1(                       	github.com/derailed/k9s/internal/model1/helpers.go:57
                             github.com/derailed/k9s/internal/model1.(*WorkerPool).Add.func1(
 	github.com/derailed/k9s/internal/model1/pool.go:57
                                                                created by github.com/derailed/k9s/internal/model1.(*WorkerPool).Add in goroutine 1
                                                                                                                                                   	github.com/derailed/k9s/internal/model1/pool.go:52
```

Pasting as is, as this is what shows on the screen, a panic/stack trace with varying indentation. I've removed a bunch of hex values to try and make it a bit clearer as I assume they are memory addresses that aren't valid in diagnosing the issue.

**To Reproduce**
Steps to reproduce the behavior:

1. `:pods`
2. shift+j on pod to jump to owner (lands on statefulset view)
3. inspect yaml of stateful set and it also has an ownerreferences value (in this case is a custom resource)
4. shift+j on statefulset
  - Check uid and name of owner reference that the owner resource does exist
6. Panic stacktrace and only way to exit is use tmux `kill-pane`

**Historical Documents**
Stack trace above. No other logs I could find

**Expected behavior**
- Jump to custom resource owner of stateful set successfully

**Screenshots**
Screenshot would just be the stacktrace as above

**Versions (please complete the following information):**

- OS: macOS
- K9s: v0.50.16
- K8s: 1.32.9

--- a/internal/render/cust_cols.go
+++ b/internal/render/cust_cols.go
@@ -172,6 +172,14 @@ func hydrate(o runtime.Object, cc ColumnSpecs, parsers []*jsonpath.JSONPath, rh 
 			continue
 		}
 
+		if o == nil {
+			cols[idx] = RenderedCol{
+				Header: cc[idx].Header,
+				Value:  NAValue,
+			}
+			continue
+		}
+
 		var (
 			vals [][]reflect.Value
 			err  error

--- a/internal/render/cust_cols_test.go
+++ b/internal/render/cust_cols_test.go
@@ -11,6 +11,7 @@ import (
 	"github.com/derailed/k9s/internal/model1"
 	"github.com/derailed/tview"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"k8s.io/client-go/util/jsonpath"
 )
 
@@ -246,7 +247,7 @@ func TestHydrateNilObject(t *testing.T) {
 
 	parser := jsonpath.New(fmt.Sprintf("column%d", 0)).AllowMissingKeys(true)
 	err := parser.Parse("{.metadata.name}")
-	assert.NoError(t, err)
+	require.NoError(t, err)
 
 	parsers := []*jsonpath.JSONPath{parser}
 	rh := model1.Header{
@@ -258,7 +259,7 @@ func TestHydrateNilObject(t *testing.T) {
 
 	// Test with nil object - should not panic
 	cols, err := hydrate(nil, cc, parsers, rh, row)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Len(t, cols, 1)
 	assert.Equal(t, NAValue, cols[0].Value)
 }

